pages/manage/productions/create.vue

```diff
@@ -3,86 +3,105 @@
-          :key="suggestedEvent['@id']"
-          :type="getEventType(suggestedEvent.terms)"
-          :title="suggestedEvent.name[locale]"
-          :image-url="suggestedEvent.image"
-          :production-name="
-            suggestedEvent.production ? suggestedEvent.production.title : ''
-          "
-          :description="
-            suggestedEvent.description ? suggestedEvent.description[locale] : ''
-          "
-        />
-      </section>
-
-      <section
-        v-if="suggestedEventsWithProduction.length < 2"
-        class="production-name-container"
-      >
-        <label for="production-name">
-          {{ $t('productions.production_name') }}
-        </label>
-        <b-input
-          id="production-name"
-          v-model="productionName"
-          autocomplete="off"
-          :disabled="suggestedEventsWithProduction.length > 0"
-          @input="handleInputProductionName"
-          @focus="handleFocusProductionName"
-          @blur="handleBlurProductionName"
-        />
+      </p>-->
+      <div v-if="suggestedEvents.length > 0">
+        <section class="events-container">
+          <event
+            v-for="suggestedEvent in suggestedEvents"
+            :id="parseEventId(suggestedEvent['@id'])"
+            :key="suggestedEvent['@id']"
+            :type="getEventType(suggestedEvent.terms)"
+            :title="suggestedEvent.name[locale]"
+            :image-url="suggestedEvent.image"
+            :production-name="
+              suggestedEvent.production ? suggestedEvent.production.title : ''
+            "
+            :description="
+              suggestedEvent.description
+                ? suggestedEvent.description[locale]
+                : ''
+            "
+          />
+        </section>
         <section
-          v-show="showSuggestedProductions"
-          class="container-table-production-name"
+          v-if="availableProductions.length < 2"
+          class="production-name-container"
         >
-          <table class="table table-hover">
-            <tbody v-if="!isLoadingSuggestedProductions">
-              <tr
-                v-for="suggestedProduction in suggestedProductions"
-                :key="suggestedProduction.production_id"
-              >
-                <td
-                  @click="
-                    handleClickProductionName(suggestedProduction.production_id)
-                  "
+          <label for="production-name">{{
+            $t('productions.production_name')
+          }}</label>
+          <b-input
+            id="production-name"
+            v-model="productionName"
+            autocomplete="off"
+            :disabled="availableProductions.length > 0"
+            @input="handleInputProductionName"
+            @focus="handleFocusProductionName"
+          />
+          <section
+            v-show="showSuggestedProductions"
+            class="container-table-production-name"
+          >
+            <table class="table table-hover">
```

Comment:
Why are we using a table here? Isn't it just a list?

Reply:
not new to this PR, but idd we can change this. I'll do this in another PR focussing on the semantics a bit (or when we tackle the blur/focus bug)

Reply:
Okay. I just noticed that the autocomplete is also not navigatable by keyboard. Other autocomplete fields in the AngularJS pages do have this (for example the field for entering the location of a new event). And it's not uncommon for users to pick a result with the arrow keys and enter key when they're already typing anyway.

Maybe we should look into a 3rd party component to handle this autocomplete instead? Like https://mattzollinhofer.github.io/vue-typeahead-bootstrap-docs/ (Not to be confused with `vue-bootstrap-typeahead` which the one I linked is a fork of, because the original did not support keyboard navigation either.)

Reply:
This is the github: https://github.com/mattzollinhofer/vue-typeahead-bootstrap/
And npm: https://www.npmjs.com/package/vue-typeahead-bootstrap

Reply:
Yeah that was my preferred solution as well, but it's not really in the scope of this PR.

Reply:
There is even an option to do it with Bootstrap, but that didn't yield the wanted results.

Reply:
Okay, can we pick that up in the PR focusing on semantics as well then?